airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/products_report.py

```diff
@@ -264,6 +282,10 @@ def _get_init_report_body(self, report_date: str, record_type: str, profile):
                 metrics_list = copy(metrics_list)
                 metrics_list.remove("sku")
 
+        if RecordType.PRODUCTADS == record_type and profile.accountInfo.type != "seller":
```

Comment:
Add a comment on why this `if` statement is needed. Explain what happens if it's missing and what happens when `profile.accountInfo.type` is other than `seller`. In addition, add a link to relevant documentation on this behavior if available.

Reply:
@monai - Added comment